hbase-server/src/main/java/org/apache/hadoop/hbase/master/cleaner/DirScanPool.java

```diff
@@ -73,8 +82,8 @@ synchronized void latchCountDown() {
     notifyAll();
   }
 
-  synchronized void execute(ForkJoinTask<?> task) {
-    pool.execute(task);
+  synchronized void runAsync(Runnable runnable) {
+    CompletableFuture.runAsync(runnable, pool);
```

Comment:
Emmm, what is this used for? The method will return a CompletableFuture but we just ignore it? Then what is the difference comparing to just call pool.execute?

Reply:
Fine,  can just use the pool#execute now.  plan to use the CompletableFuture of runAsync to get the final result befoe.